code_search/code-search.ipynb

```diff
@@ -196,7 +202,7 @@
     "cd kubeflow\n",
     "\n",
     "# Update Ksonnet to point to the Kubernetes Cluster\n",
-    "ks env add code-search --context $(kubectl config current-context)\n",
+    "ks env add code-search\n",
     "\n",
     "# Update Ksonnet to use the namespace where kubeflow is deployed. By default it's 'kubeflow'\n",
     "ks env set code-search --namespace=kubeflow\n",
```

Comment:
maybe define a env variable for the environment and used for all places inside this guideline?

Reply:
I think that makes sense but I figured I'd leave that for a follow on PR; since the notebook wasn't using an env variable.